addon/components/sortable-items.js

```diff
@@ -114,7 +114,11 @@ var SortableItems = Ember.Component.extend({
     An item is dropped into the list from another list
   */
   _onAdd: function(evt) {
-    this._sendOutAction('onAddAction', evt);
+    Ember.run(this, function() {
+      var collection = this.get('itemCollection');
```

Comment:
It turns out that at this point, `itemCollection` is the new list, not the old list.